src/features/build_features.py

```diff
@@ -1,65 +1,85 @@
 import pandas as pd
 from pyalgotrade import strategy
 from pyalgotrade.technical import ma, linreg
-from pyalgotrade.dataseries import SequenceDataSeries
 
 
 class BuildFeatures(strategy.BacktestingStrategy):
+    """
+    Construct features (and labels for training) of stock market indicators from daily CSV files
+    using various stock analysis techniques
+
+    Example usage:
+    b = BuildFeatures()
+    X, y = b.run()
+    """
+
     def __init__(self, feed, instrument):
         strategy.BacktestingStrategy.__init__(self, feed)
-        self.__instrument = instrument
-        self.__sma = ma.SMA(feed[instrument].getAdjCloseDataSeries(), 15)
-        self.__ema = ma.EMA(feed[instrument].getAdjCloseDataSeries(), 15)
+        self.instrument = instrument
+        self.sma = ma.SMA(feed[instrument].getAdjCloseDataSeries(), 15)
```

Comment:
can we make `15` a parameter on constructor - and the other magic numbers s
```suggestion
        self.sma = ma.SMA(feed[instrument].getAdjCloseDataSeries(), sma_window)
```

Reply:
ditto for other `15` locations